addon/-private/utils/element.js

```diff
@@ -30,13 +30,45 @@ export function closest(el, selector) {
   return null;
 }
 
+/*
+ * Calculate the scale of difference between an element's logical height
+ * (the offsetHeight or getComputedStyle height) compared to its rendered
+ * height (via getBoundingClientRect).
+ *
+ * Note that there are some interesting edge cases to consider around
+ * these APIs.
+ *
+ *   - `getComputedStyle` returns a string from styles in pixels. For
+ *     example `48.23px`. The precision of this API in Chrome seems to
+ *     be 3 decimal places. Additionally, there can be unexpected values
+ *     such as `auto`. Finally, this API may be slow compared to other
+ *     DOM API calls.
+ *   - `offsetHeight` always returns in integer. If the height of an
+ *     element is at float precision, then the value is rounded. This
+ *     makes it at best useful as a fallback.
+ *
+ */
 export function getScale(element) {
   let rect = element.getBoundingClientRect();
+  let renderedHeight = rect.height;
 
-  if (element.offsetHeight === rect.height || rect.height === 0) {
+  if (renderedHeight === 0) {
+    return 1;
+  }
+
+  let computedHeightStyleValue = window.getComputedStyle(element).height;
+  let computedHeightInPixels = Number(
+    computedHeightStyleValue.substring(0, computedHeightStyleValue.length - 2)
```

Comment:
can we use `parseFloat` here, maybe ?

also, how would this handle the `auto`, `inherit`, `initial` and similar non-pixel string values ?

Reply:
`parseFloat` docs suggest `Number(string)` as having stricter parsing: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseFloat I don't think there is a reason to use `parseFloat` over `Number`.

`isNaN(Number('a')) === true`, hence the `isNaN` check on the next line. I can leave a comment about this.